app/instructionalSupport/studentSupportCallForm/services/studentService.js

```diff
@@ -13,9 +13,21 @@ instructionalSupportApp.factory("supportStaffFormService", this.supportStaffForm
 
 			return deferred.promise;
 		},
-		addStudentPreference: function(preference) {
+		getDwActivitiesByCrn: function (crn, termCode) {
```

Comment:
Do we have a DW Service already? The $http logic should be in there so dwUrl isn't needed in studentService.js.

Reply:
We don't have one already, but thats a sensible idea, I'll make one